### ISSUE TITLE
feat: allow customisation of DoH path

### ISSUE THREAD

**Issue by adrianmace** [0xERR0R/blocky#1545]
**As a** customer,
**I want** to be able to customise the path at which the server responds to DoH queries,
**So that** I can reduce the number of bots querying my server while it is exposed publicly.

---

Use Case: I would like to run an instance of blocky publicly for my mobile devices to use while roaming. This instance would be an IPv6-only instance to reduce the amount of botnet/spam, but to further reduce the amount of malicious requests I'd like to customise the path too.

---

TL;DR Make this constant user-customisable:
https://github.com/0xERR0R/blocky/blob/ddebd3450c71b88115f8b32f698fea1f643a4b48/server/server_endpoints.go#L68

**Comment by 0xERR0R** [comment]
Was implemented in https://github.com/0xERR0R/blocky/pull/1843

will add logging and validation
